mechanics/WORKING-GROUP-PROCESSES.md

```diff
@@ -42,9 +42,18 @@ the number of working groups low in order to keep things manageable.
 
 ## Proposing a new working group
 
-If you’ve identified a substantial architectural area which would benefit from
-long-lived, concerted and focused design, then you should consider creating a
-new working group. To do so, you need to:
+Generally, working groups should be founded once sufficient community interest
+and discussion has occurred that there is general agreement that:
+
+1. The problem space is interesting and is worth long-term investment by Knative
+
+1. The problem is not well-covered by any of the existing chartered WGs
```

Comment:
I think the way I'd put this is: "WG proposals shouldn't be a surprise to interested parts of the community".

By the time it reaches a WG proposal, it should be well socialized and the right parties should be pulled in to the formation.  I don't think we need consensus or a solution, but I think for inclusion upstream we want to be better about things having broad support from the community.

Donation (per Mike's comment above) should be limited to cases where there is early consensus on one particular approach.  I also think donation can happen after formation if that consensus arrives later as part of the WG process.

Reply:
I added a paragraph warning against "code dumps" as the start of a WG:

> Note that "code dumps" are unlikely to be accepted as the beginning of a working
> group, though they can be useful to start a conversation. For most topics that
> meet the bar for a new WG, it's likely that other participants in the community
> have ideas and opinions, and up-front conversations can help to focus the shape
> and scope of the solution space.